grease-cli/tests/refine/neg/networking/test.c

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) Galois, Inc. 2025 */
+
+// Test that the overrides for networking-related functions work as expected.
+
+// all: flags {"--symbol", "test"}
+// all: flags {"--fs-root", "tests/refine/neg/extra/server-fs"}
+// all: go(prog)
+
+#include <arpa/inet.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+void test(void) {
+  int sock_fd = socket(AF_INET, SOCK_STREAM, 0);
+  if (sock_fd == -1) {
+    return;
+  }
+
+  struct sockaddr_in server;
+  server.sin_family = AF_INET;
+  server.sin_addr.s_addr = INADDR_ANY;
+  // NB: the number 5000 becomes 34835 when converted from host byte order
+  // to network byte order, which is why the corresponding directory in the
+  // symbolic filesystem is
+  // tests/filesystems/server-inet/root/network/AF_INET/SOCK_STREAM/34835.
```

Comment:
Does this path need to updated? It looks like it was copied over from a different project that used "`server-inet`" in the path (which GREASE isn't doing, AFAICT).